include/NovelRT/Timing/Timestamp.h

```diff
@@ -0,0 +1,33 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_TIMING_DELTATIMESTAMP_H
+#define NOVELRT_TIMING_DELTATIMESTAMP_H
+
+#ifndef NOVELRT_H
+#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
+#endif
+
+namespace NovelRT::Timing {
+  class Timestamp {
+  private:
+    uint64_t _ticks;
```

Comment:
I removed the `const` qualifier from here due to a deleted function which meant I couldn't assign it to a private field, lol. Woops.

Reply:
Could you elaborate as to which function is preventing this from being effectively readonly?

Reply:
in the tests, `_timeStamp = TimeStamp(0)` means that I'd need to define a custom `=` operator, which I didn't feel like doing, especially when the API surface effectively makes this `const` anyway.

Reply:
Ah right, because C++ assignment for `const` isn't the same as C# with `readonly`.

I don't remember the right syntax for this...

Reply:
I don't either! 😄